marketplace/commercetools-sku/src/index.js

```diff
@@ -14,6 +14,34 @@ function makeCTA(fieldType) {
   return fieldType === 'Array' ? 'Select products' : 'Select a product';
 }
 
+function validateParameters(parameters) {
+  if (parameters.projectKey.length < 1) {
+    return 'Provide your Commercetools project key.';
+  }
+
+  if (parameters.clientId.length < 1) {
+    return 'Provide your Commercetools client ID.';
+  }
+
+  if (parameters.clientSecret.length < 1) {
+    return 'Provide your Commercetools client secret.';
+  }
+
+  if (parameters.apiEndpoint.length < 1) {
+    return 'Provide the Commercetools API endpoint.';
```

Comment:
The messages above say "your". Is this endpoint and the one below going to be static for all user? If so, can't we just hardcode it in?

Reply:
I'm not clear on this either tbh, I just kept it because it was like this in Rouven's PR, but we should clear this up with Ivo from CT before merging.

Reply:
@davidfateh so the update is that they can differ.. based on region (e.g. EU vs US), I'd say we can leave the fields as is for now, but I think it also makes more sense to keep "the" instead of "your" since it's not really individual?

Reply:
Sounds good.